notebook/app.py

```diff
@@ -32,11 +32,14 @@
 from notebook_shim.shim import NotebookConfigShimMixin  # type:ignore[import]
 from tornado import web
 from traitlets import Bool, Unicode, default
+from traitlets.config.loader import Config
 
 from ._version import __version__
 
 HERE = os.path.dirname(__file__)
 
+Flags = t.Dict[t.Union[str, t.Tuple[str, ...]], t.Tuple[t.Union[t.Dict[str, t.Any], Config], str]]
```

Comment:
cc @blink1073 

Wondering if there would be a better / simpler way to handle this in downstream apps?

Reply:
ok thanks!